components/brave_wallet_ui/common/async/lib.ts

```diff
@@ -274,6 +275,17 @@ export const getAllBuyAssets = async (): Promise<{
   return results
 }
 
+export function getFilecoinKeyringIdFromNetwork (network: BraveWallet.NetworkInfo) {
```

Comment:
Can we move this method to `brave_wallet_ui/utils/network-utils.ts` along with a unit test?